app/src/main/java/org/mifos/mobile/ui/activities/PassCodeActivity.kt

```diff
@@ -79,6 +81,14 @@ class PassCodeActivity : MifosPassCodeActivity() {
         )
     }
 
+    private fun setBackgroundColor(){
+        val nestedScrollView = findViewById<NestedScrollView>(R.id.cl_rootview)
+        val colorBackground = android.R.attr.colorBackground
+        val typedValue = TypedValue()
+        theme.resolveAttribute(colorBackground, typedValue, true)
```

Comment:
Instead of creating `colorBackground` variable, you could directly use `android.R.attr.colorBackground` in here